### ISSUE TITLE
reinforce_learn_qnet ArgumentError when running through command line 

### ISSUE THREAD

**Issue by sid-sundrani** [PyTorchLightning/pytorch-lightning#5382]
## 🐛 Bug
I'm running `reinforce_learn_qnet` in pl_examples through the command line. When I run `python reinforce_learn_Qnet.py`, I get an argument conflicting options error: 
```
Traceback (most recent call last):
  File "reinforce_learn_Qnet.py", line 428, in <module>
    parser = DQNLightning.add_model_specific_args(parser)
  File "reinforce_learn_Qnet.py", line 387, in add_model_specific_args
    parser = argparse.ArgumentParser(parents=[parent_parser])
  File "C:\Users\Sidhant\AppData\Local\Programs\Miniconda3\envs\torch\lib\argparse.py", line 1678, in __init__
    self._add_container_actions(parent)
  File "C:\Users\Sidhant\AppData\Local\Programs\Miniconda3\envs\torch\lib\argparse.py", line 1447, in _add_container_actions
    group_map.get(action, self)._add_action(action)
  File "C:\Users\Sidhant\AppData\Local\Programs\Miniconda3\envs\torch\lib\argparse.py", line 1577, in _add_action
    action = super(_ArgumentGroup, self)._add_action(action)
  File "C:\Users\Sidhant\AppData\Local\Programs\Miniconda3\envs\torch\lib\argparse.py", line 1387, in _add_action
    self._check_conflict(action)
  File "C:\Users\Sidhant\AppData\Local\Programs\Miniconda3\envs\torch\lib\argparse.py", line 1526, in _check_conflict
    conflict_handler(action, confl_optionals)
  File "C:\Users\Sidhant\AppData\Local\Programs\Miniconda3\envs\torch\lib\argparse.py", line 1535, in _handle_conflict_error
    raise ArgumentError(action, message % conflict_string)
argparse.ArgumentError: argument -h/--help: conflicting option strings: -h, --help
```

imagenet.py in pl_examples runs perfectly fine though through the command line for me. The script passes a add_help=False argument `ArgumentParser(add_help=False)`. When I add the same add_help argument to [reinforce_learn_qnet](https://github.com/PyTorchLightning/pytorch-lightning/blob/019e4ff8cddadfde4d3bd48b4e8b8d294950ca2a/pl_examples/domain_templates/reinforce_learn_Qnet.py#L427) it works without the argument error for me 

### To Reproduce

https://colab.research.google.com/drive/1WKmHMmpXpzd75i4rqyMAKO_HHo_tA5oK?usp=sharing
```
! pip install pytorch-lightning
! git clone https://github.com/PyTorchLightning/pytorch-lightning.git
! python pytorch-lightning/pl_examples/domain_templates/reinforce_learn_Qnet.py
```

### Environment
```
 - PyTorch Version (e.g., 1.0): 1.6.0
 - OS (e.g., Linux): Windows 10
 - How you installed PyTorch (`conda`, `pip`, source): Conda 
 - Python version: 3.7.3

**Comment by Borda** [comment]
seems to be duplicate to #5381

**Comment by sid-sundrani** [comment]
Hm is it duplicate? I fixed #5381 and it didn't fix this. I think this issue is different

**Comment by sid-sundrani** [comment]
But actually yes, after I fixed this, it still didn't run through the command line until I also fixed #5381
